src/popgun.scss

```diff
@@ -190,10 +190,12 @@ $boxShadowPop: 1px 2px 2px rgba(#1a333d, 0.15);
   overflow: visible;
   white-space: normal;
 
-  transition: top .25s ease-out, left .25s ease-out;
+  -webkit-transition: top .25s ease-in-out, left .25s ease-in-out;
+  transition: top .25s ease-in-out, left .25s ease-in-out;
```

Comment:
I don't particularly care that this changed from `ease-out` to `ease-in-out`, but I just wanted to hear the reasoning.